scripts/globals/items/astral_cube.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------------
+-- ID: 6413
+-- Item: Astral Cube
+-- Item Effect: Grant Astral cube key item
+-----------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/keyitems");
+require("scripts/globals/msg");
+
+local keyItemId = dsp.ki.ASTRAL_CUBE;
+
+function onItemCheck(target)
+    if (target:hasKeyItem(keyItemId)) then
```

Comment:
we're killing parentheses in lua (they're optional)

`if (thing) then` -> `if thing then`

Reply:
I've sent them to a farm upstate.